adapters/bidder.go

```diff
@@ -39,6 +39,20 @@ type Bidder interface {
 	MakeBids(internalRequest *openrtb.BidRequest, externalRequest *RequestData, response *ResponseData) (*BidderResponse, []error)
 }
 
+// TimeoutBidder is used to identify bidders that support timeout notifications.
+type TimeoutBidder interface {
+	MakeRequests(request *openrtb.BidRequest, reqInfo *ExtraRequestInfo) ([]*RequestData, []error)
+	MakeBids(internalRequest *openrtb.BidRequest, externalRequest *RequestData, response *ResponseData) (*BidderResponse, []error)
```

Comment:
We should be able to get this same behavior of inheriting the standard `bidder` interface methods by embedding the bidder interface here. That way we won't have to manage the method signatures in two places.

Reply:
Done